java-tools/OsmAndServer/src/main/java/net/osmand/server/api/searchtest/MapDataObjectFinder.java

```diff
@@ -83,10 +89,11 @@ public Result[] find(List<SearchResult> searchResults, long datasetId) {
 		return new Result[] {firstResult, null};
 	}
 
-	public Result getMapDataObject(SearchResult result, long expectedOsmId, int place) {
+	public Result getMapDataObject(SearchResult result, long expectedOsmId, int place, Map<String, Object> row) {
```

Comment:
1. We always read file again ? this is very slow, we don't even check if we have already id or not

Reply:
2. Row is only used by_tag which not needed